dags/analyzer_helper/tests/unit/test_transform_raw_members.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+
+from analyzer_helper.discourse.transform_raw_members import (
+    TransformRawMembers,
+)
+
+class TestDiscordTransformRawMembers(unittest.TestCase):
+    def test_transform_empty_list(self):
+        """
+        Tests that transform returns an empty list for an empty members list
+        """
+        transformer = TransformRawMembers()
+        result = transformer.transform([])
+        self.assertEqual(result, [])
+
+    def test_transform_single_member(self):
+        """
+        Tests that transform correctly transforms a single member
+        """
+        transformer = TransformRawMembers()
+        raw_member = {
+            "id": 85149,
+            "avatar": "avatar2",
+            "createdAt": "2023-07-02",
+            "badgeIds": ["badge2"],
+        }
+
+        expected_result = [
+            {
+                "id": 85149,
```

Comment:
Please make the id of rawmembers data to be string rather than integer. When saving the analytics results using the integer user ids (int64), we're facing issues for MongoDB saving.